jmclient/jmclient/storage.py

```diff
@@ -0,0 +1,333 @@
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import os
+import shutil
+import bencoder
+import pyaes
+import random
+from hashlib import sha256
+from argon2 import low_level
+
+
+# FIXME: move this to .support
+def get_random_bytes(num_bytes, cryptographically_secure=False):
+    if cryptographically_secure:
+        generator = random.SystemRandom()
```

Comment:
For convenience, I'd recommend adding a comment that this is using os.urandom ... only because it's the *first* thing that anyone wanting to review would want to check in any Bitcoin software they use :) As for the default False for `cryptographically_secure`, it's fine, I realise you chose it based on the fact that only one usage doesn't choose that option.